<launch>
  <node name="gamepad_publisher" pkg="gamepad" type="node" >
    <param name="frequency" type="double" value="50.0" />
    <param name="topic" type="string" value="/gamepad/data" />
    <param name="gamepad_path" type="string" value="/dev/input/js0" />
    <param name="buffer_size" type="int" value="10" />
  </node>
</launch>